stories/index.stories.js

```diff
@@ -99,49 +99,56 @@ storiesOf('Breadcrumb', module)
   )));
 
 storiesOf('Buttons', module)
-  .add('Primary Button', withInfo('')(() => (
+  .add('Primary Button', withInfo('Buttons set an action in motion. Instead of using “Yes” or “Okay”, it’s better to use verbs that explain what the triggered action will be. For example, “Create new”, “Discard draft”, etc. Use the primary button when it’s the main action on a page/table. Use the smaller one in case of limited space.')(() => (
     <div>
       <Button bsStyle="primary" onClick={action('clicked')}>{'Primary Button'}</Button>
       <Button bsSize="small" bsStyle="primary" onClick={action('clicked')}>{'Primary Small Button'}</Button>
     </div>
   )))
 
-  .add('Secondary Button', withInfo('')(() => (
+  .add('Default Button', withInfo('Default buttons are used for secondary actions. They’re usually paired with a primary button to provide an alternate choice.')(() => (
```

Comment:
Bootstrap v4 has a dedicated "secondary" style (and with that I'm not sure we'd ever use a default button), so I removed this in https://github.com/metabrainz/design-system/pull/18. If we do think a button with no set style is useful we could add it back, but it'd need a different description at least.